geode-docs/reference/topics/gemfire_properties.html.md.erb

```diff
@@ -117,7 +117,7 @@ Valid values are in the range 0...2147483647</td>
 </tr>
 <tr>
 <td>cluster-configuration-dir</td>
-<td>This property specifies the directory in which the cluster configuration related disk-store and artifacts are stored. This property is only applicable to dedicated locators that have &quot;enable-cluster-configuration&quot; set to true.</td>
+<td><b>Deprecated.</b> Use import/export clutser-configuration command instead. This property specifies the directory in which the cluster configuration related disk-store and artifacts are stored. This property is only applicable to dedicated locators that have &quot;enable-cluster-configuration&quot; set to true.</td>
```

Comment:
I only looked at the docs portion of this PR.

1. Please change "clutser" to "cluster"
2. Please add "gfsh" in to this sentence, since we're not substituting a different property, and I think that you mean to use a gfsh command instead.  Something like "Use the gfsh import or export cluster-configuration command instead."

Reply:
I also recommend mentioning that it currently does nothing. We've done this in the past for APIs that do nothing -- deprecate and document that it does nothing and what the user should do instead.